src/checkpt2/checkpt2.css

```diff
@@ -0,0 +1,530 @@
+}
+#events .article small {
+  color: #888;
+}
+#events a:hover .article small {
+  color: #fff;
+}
+/* EVENTS PAGES */
+#event-page {
+  background: #000;
+  color: #fff;
+}
+#event-page nav {
+  background: #333;
+}
+#event-page main {
+  padding: 4rem 0;
+  display: flex;
+}
+#event-page main h2 {
+  margin-left: 5rem;
+  font-size: 4rem;
+  padding-bottom: 1rem;
+  text-shadow: 2px 3px var(--primary-color);
+}
+#event-page main img {
+  margin-left: 5rem;
+  height: 500px;
+  max-width: 600px;
+  box-shadow: 5px 4px 10px var(--primary-color);
+}
+#event-page main div:last-child {
+  padding: 0 4rem;
+  align-self: center;
+  font-size: 1.4rem;
+}
+#event-page main div:last-child p {
+  padding-bottom: 1.4rem;
+}
+
+/* ABOUT */
+#about {
+  display: flex;
+  justify-content: flex-end;
+  height: 700px;
+  background: var(--primary-color);
+}
+#about div:first-child {
+  background: url('./img/about.jpg') no-repeat center center/cover;
+  width: 100%;
+  border-bottom-right-radius: 50%;
+}
+#about div:first-child h2 {
+  font-size: 5rem;
+  font-family: 'Roboto Condensed', sans-serif;
+  text-decoration: underline;
+  color: #fff;
+  text-align: end;
+  text-shadow: 1px 2px #333;
+  margin: 3rem 6rem 0 0;
+}
+#about div:last-child {
+  width: 80%;
+  padding: 4rem 1.5rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+#about div:last-child p {
+  font-size: 1.3rem;
+  font-weight: 800;
+  margin-bottom: 2rem;
+  max-width: 600px;
+}
+#about video {
+  max-width: 600px;
+  border: 2px solid #333;
+}
+
+/* CONTACT */
+#contact {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 4rem 0;
+}
+#contact div:first-child {
+  border-bottom: 1px dotted;
+  padding-bottom: 1rem;
+}
+#contact h3 {
+  text-align: center;
+  font-size: 2rem;
+  font-weight: 800;
+  margin-bottom: 1.25rem;
+}
+#contact p {
+  text-align: center;
+  font-size: 1.2rem;
+  margin-bottom: 1.25rem;
+}
+#contact h2 {
+  font-size: 3.3rem;
+  text-align: center;
+  font-weight: 800;
+  margin: 3rem 0;
+}
+#contact form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+#contact form label {
+  font-size: 1.3rem;
+  margin-bottom: 0.5rem;
+}
+#contact form input,
+#contact form textarea {
+  width: 100%;
+  font-size: 1.4rem;
+  margin-bottom: 0.5rem;
+  border: 1px solid #333;
+}
+#contact form input[type="submit"] {
+  cursor: pointer;
+  background-color: #333;
+  color: #fff;
+  width: 200px;
+  padding: 0.75rem;
+  border-radius: 35px;
+  margin-top: 0.5rem;
+}
+#contact form input[type="submit"]:hover {
+  color: var(--primary-color);
+}
+
+/* FOOTER */
+footer {
+  padding: 2.5rem 0 1rem 0;
+  background: #333;
+  color: #fff;
+}
+footer .social {
+  display: flex;
+  justify-content: center;
+}
+footer .social a {
+  font-size: 1.2rem;
+  color: #fff;
+  background: var(--primary-color);
+  border-radius: 50%;
+  margin: 1rem;
+  padding: 1rem;
+}
```

Comment:
these orange circles aren't round and look a lil weird. set a matching height and width and center the i tag with flexbox.